test/ssc_test/cmod_utilityrate5_test.cpp

```diff
@@ -267,7 +267,7 @@ TEST(cmod_utilityrate5_eqns, Test_Residential_net_metering_credits_in_may_with_r
     EXPECT_NEAR(0, may_dollars, 0.1);
 
     float june_dollars = excess_dollars[credit_month + 1];
-    EXPECT_NEAR(50.28, june_dollars, 0.1);
+    EXPECT_NEAR(11.37, june_dollars, 0.1);
```

Comment:
Does the change show the difference between the true up and the rollover when happening mid-year?

Reply:
The previous value here was for the "cut a check a month late" code, so the old value was the full bill for June. The new code applies the credit to the bill, so the June value is reduced. To compare with the "cut a check in the true up month" rate, see Test_Residential_net_metering_credits_in_may